### PR TITLE
Chapter 7 pass

### PR DESCRIPTION
**Pull request by krzychukula**
Quotes
Grammar

### CODE REVIEW

File: manuscript/ch7.md [hunk 1]
```diff
@@ -166,12 +166,12 @@ app({
...
 });
 ```
-Because you followed the same naming convention for ```action``` and ```url``` it should be just a matter of switching ```url``` and ```WebSocketListen``` to ```EventSourceListen```.
+Because you followed the same naming convention for `action` and `url`, it should be just a matter of switching `url` and `WebSocketListen` to `EventSourceListen`.
 
 Test your application. It should work the same way as the WebSocket version, but without switching to a different protocol. 
 
 Diagnosing problems with SSE:
-* make sure the ```eventsource``` connection-type was established
+* make sure the `EventSource` connection-type was established
```
Comment (krzychukula) [inline manuscript/ch7.md:174]: The lower case seemed like a typo.

Comment (kwasniew) [reply manuscript/ch7.md:174]: this is what DevTools reports. It uses eventsource in the network tab.

File: manuscript/ch7.md [hunk 2]
```diff
@@ -17,17 +17,17 @@ To build intuition about subscriptions look at different even sources that fit t
 
 What they have in common is a long-lived nature of the underlying event source. 
 
-Note: I'll refer to short-live effects as just effects and to long-lived effects as subscriptions.
+Note: I'll refer to short-lived effects as just effects and too long-lived effects as subscriptions.
```
Comment (kwasniew) [inline manuscript/ch7.md:20]: too -> to. It was correct

File: manuscript/ch7.md [hunk 3]
```diff
@@ -124,12 +124,12 @@ Start with the same signature for the subscription definition:
...
 ```
 
-The browser API for SSE is called the ```EventSource```:
+The browser API for SSE is called the `EventSource`:
 ```js
 const es = new EventSource("https://hyperapp-api.herokuapp.com/api/event/post");
 es.addEventListener("message", event => /* handle event with a data field */)
 ```
-```EventSource``` is a regular event emitter similar e.g. to a clickable button.
+`EventSource` is a regular event emitter similar, e.g. to a clickable button.
```
Comment (kwasniew) [inline manuscript/ch7.md:132]: that's a strange usage of comma. Are you sure it's correct?

Comment (krzychukula) [reply manuscript/ch7.md:132]: Look good to me, but the sentence is a little weird regardless. Maybe there is a way of making it simpler? 
```suggestion
`EventSource` is a regular event emitter similar to a clickable button.
```

File: manuscript/ch7.md (hunk 2 above)
Comment (krzychukula) [reply manuscript/ch7.md]: Removed entirely. What do you think?

**Pull request merged**